log/experimental_level/level.go

```diff
@@ -147,10 +63,87 @@ func (l *logger) Log(keyvals ...interface{}) error {
 	return l.next.Log(keyvals...)
 }
 
-func makeSet(a []string) map[string]struct{} {
-	m := make(map[string]struct{}, len(a))
-	for _, s := range a {
-		m[s] = struct{}{}
-	}
-	return m
+// Option sets a parameter for the leveled logger.
+type Option func(*logger)
+
+// AllowAll is an alias for AllowDebugAndAbove.
+func AllowAll() Option {
+	return AllowDebugAndAbove()
+}
+
+// AllowDebugAndAbove allows all of the four default log levels.
+func AllowDebugAndAbove() Option {
+	return allowed(levelDebug | levelInfo | levelWarn | levelError)
+}
```

Comment:
Hmm. My philosophy tends to be that any code change that deviates from a naive, intuitive implementation needs a degree of justification proportionate to that deviation. There is a lot of subjectivity built in to that criteria, unfortunately. To my subjective eyes, bitmasking levels in this way is enough of a headscratcher that I'd want to see a much clearer win than a 3% performance improvement. With that objection raised, I defer to Chris' final call.